### ISSUE TITLE
[BUG]: Missing dashboard name in new Reveal version

### ISSUE THREAD

**Issue by dwilches** [RevealBi/Reveal.Sdk#197]
### SDK Version

1.6.6

### Client Framework

Angular

### Server Platform

Java

### Operating System

Linux

### Description

Today we upgraded from Reveal 1.6.1 to 1.6.6 and while in Edit mode, our dashboards stopped looking like this:

![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/c97269fa-bbfc-462d-8203-61c01ef51ade)

And started looking like this:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/955d68f8-6d42-4f80-ac55-2dfea4037568)

So now the name of the dashboard is not visible, and in its place, we have an empty box and a text area:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/1cd7d48a-955d-4762-a312-1d59ad440616)

**Comment by luispandolfi** [comment]
Hi @dwilches, does it happens for all the dashboards?
So, let me check if I understood:
- if you load a dashboard file in 1.6.1 it shows the title,
- but, if you load the same dashboard file in 1.6.6 the title is not shown

Could you share the dashboard file?

Regarding the dashboard description, yes that is a new feature. If you do not want to see that you can hide from the RevealView with the property ShowDescription.

**Comment by dwilches** [comment]
@luispandolfi do you have a private way I can share the dashboard file? It contains information I'd rather not share with the world

**Comment by dwilches** [comment]
@luispandolfi Correct, the images posted above were for the exact same dashboard, as seen in 1.6.1 and 1.6.6

I just gave a try to `showDescription = false` and it hides the grey field at the bottom. The name of the dashboard is still missing though:

![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/9547a030-2219-4df5-8480-aaca39730b1e)

I just tried to create a new dashboard in 1.6.6 to see if it looked better, but notice the name of the dashboard can't be changed even if it's a completely new:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/938efb7c-c194-43c3-a37a-9f1eecf01555)

This is what I would have normally see with 1.6.1 for a new dashboard:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/ba555bc1-a442-4a7f-a283-358d3e74f2ba)

Here is the dashboard generated by 1.6.6, which exhibits the same issue of the missing title:
```
UEsDBBQACAgIAMOQp1gAAAAAAAAAAAAAAAAOAAAARGFzaGJvYXJkLmpzb259UttuwjAMfecrKp6p1DK0jb3BgA2JXbR2DGmakNsYFilNqiQVN/HvS1IoZUx7iWQf+5xjx7uG5zXHKsYsZ6CxeectgCls2fS7wl6hxQQ2otAG0bIogQFoiEQhU1Qm/fnlkh+ULFG7hAk9b+deA9wLVmQ8yoEb7CpoHfNzvcmt4KEztlEFTqkqgNEtaCp4hFpTvrTcR9bfJfGBKx7O4nn/ZVYx1YViXOu+WP/J7QgOPfvKxptYXRp34+eYntvpMbrkGXK7qeYEF7puYSS4jujWuXhCQovsP4NHgbOVmLrhOqfSuTbFYacT1DDbaikekTHhrYRk5HKcMbElIZKwndyAf40Q+J2gHfrdsHvrJ+00TAnpJt0ETj/xKpSOJXC1EDJz4qdPL0lVTDVDs1WasOqCHLo3b3kdD0wkwKYgKZii2tm4XuvqGVfeANR3IkASJ2+2ZhWnKFU5clCjGlGmDeCIGvsfUEsHCB9JJq1jAQAAxwIAAFBLAQIUABQACAgIAMOQp1gfSSatYwEAAMcCAAAOAAAAAAAAAAAAAAAAAAAAAABEYXNoYm9hcmQuanNvblBLBQYAAAAAAQABADwAAACfAQAAAAA\u003d
```

**Comment by luispandolfi** [comment]
Ok, I think the rdash file is not needed for now. Probably some client side issue.
Just to confirm, are you using angular? Any other client-side framework you use with the RevealView?

**Comment by luispandolfi** [comment]
Could you share a simple sample that replicate the issue when creating a new dashboard? I tried it but worked for me.

**Comment by brianlagunas** [comment]
Hey @dwilches can you try running this sample and let me know if you see the issue? It's a 1.6.6 Angular application.

https://demo.revealbi.io/home

**Comment by dwilches** [comment]
Hi
It looks ok in that page:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/6c201c15-1c8c-40b5-a1c4-142a21a7ffbf)

I checked devTools, and I see in your page the element `#DialogTextBox` has a height of 26px:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/6dd0e464-b815-4451-9a32-4b539ab42f81)

Whereas in mine, the height is 0px:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/12877077-80aa-4229-abfa-277c6fbadbe6)

Could you share what do you use to calculate the size of that element? IT may hint into what to look at

**Comment by dwilches** [comment]
By any chance are the font-family measurements being used to calculate that height? Because of this other issue I just noticed: https://github.com/RevealBi/Reveal.Sdk/issues/206 . In Reveal 1.6.1 I see 1 such warnings, but in Reveal 1.6.6 I started seeing 2 warnings.

**Comment by dwilches** [comment]
I kept debugging a little more, and I found a CSS line that is causing the height to be considered 0px.

We have this in our global CSS:

```
input {
    transition: all 0.15s linear;
}
```

If I change it to `transition: unset` then the dashboard name starts appearing:
![image](https://github.com/RevealBi/Reveal.Sdk/assets/3975401/ca67171b-5da6-497a-969f-03f50942d380)


So I'm guessing the calculation for the input height is being done too early, before the animations are completed?

Anyways, now that I know its caused by custom CSS, I can add an override so it doesn't apply to Reveal.

Thanks.
